R/time.R

```diff
@@ -0,0 +1,197 @@
+#' Time Feature Generator
+#'
+#' `step_time` creates a *specification* of a recipe
+#'  step that will convert date-time data into one or more factor or
+#'  numeric variables.
+#'
+#' @inheritParams step_pca
+#' @inheritParams step_center
+#' @param ... One or more selector functions to choose variables
+#'  for this step. The selected variables should have class
+#'  `POSIXct` or `POSIXlt`. See [selections()] for more details.
+#' @param features A character string that includes at least one
+#'  of the following values: `ampm` (AM or PM), `hour24`, `hour12`,
+#'  `minute`,`second`, `decimal_day`.
```

Comment:
Decimal day doesn't feel that useful to me, but I do think it should be documented if you want to retain it. I didn't know what  it did until I looked at the implementation

Reply:
keeping it, added documentation